docs/syntax/html.md

```diff
@@ -38,7 +38,7 @@ Where Origami components include or output HTML, it should meet the following re
 * Those elements and attributes which are deprecated in the HTML5 spec *should* not be used:
 	- BAD: `<applet>`, `<frameset>`, `<font>`, `<link rev="">`, `<td align="right">`
 * `<iframe>` *must* not be used in markup. Iframes may be created by JavaScript.
-* Conditional comments *must not* be used in components or recommended to product developers. Components should instead rely on classes on the `body` or `html` element that indicate feature support. Component authors may require the product application to set any feature support classes supported by [Modernizr](http://modernizr.com/docs/).  Product developers may of course choose to apply those classes using conditional comments rather than using Modernizr.
+* Conditional comments *must not* be used in components or recommended to product developers. Components should instead rely on classes on the `body` or `html` element that indicate feature support. Product developers may of course choose to apply those classes using conditional comments.
```

Comment:
Do we still need to use classes in anyway to indicate support?  Should we stick with the Modernizr convention in those cases?